### ISSUE TITLE
Open, non-termed seats not shown as vacant

### ISSUE THREAD

**Issue by charlesbrandt** [City-of-Bloomington/OnBoard#49]
Ex-Officio seats are not showing up as vacant, even with no current member assigned. For example:
http://rogue.bloomington.in.gov/boards-staging/committees/members?committee_id=9

**Comment by rbdietz** [comment]
We might want to consider a way to indicate proxies for ex-officios.  For
example, the the Mayor may be the ex-officio member of a body, but he can
appoint a temp or permanent proxy.  Not a priority now, but something to
consider.

On Thu, Jan 21, 2016 at 4:34 PM, Cliff Ingham <notifications@github.com>
wrote:

> Closed #49 <https://github.com/City-of-Bloomington/OnBoard/issues/49> via
> 5d50265
> <https://github.com/City-of-Bloomington/OnBoard/commit/5d5026576eb1317e9f7518ff71f7827bb35ebef7>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/City-of-Bloomington/OnBoard/issues/49#event-522226803>
> .
>



-- 
~~~~~~~~~~~~~~~~~~~~~~~~~
Rick Dietz, Director
Information & Technology Services Dept.
City of Bloomington, Indiana
812.349.3485 (office) 812.340.7958 (cell)
dietzr@bloomington.in.gov

**Comment by inghamn** [comment]
Alternates and proxies are already in the issue tracker as a feature request.

#9
